packages/phone-number-privacy/common/src/utils/contracts.ts

```diff
@@ -3,6 +3,7 @@ import { ContractKit, newKit, newKitWithApiKey } from '@celo/contractkit'
 export interface BlockchainConfig {
   provider: string
   apiKey?: string
+  timeout_ms?: number
```

Comment:
Nit: Hate to be a huge pain but can you change this to camel case for consistency?